backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -281,7 +280,7 @@ private async void OnIsarMissionUpdate(object? sender, MqttReceivedArgs mqttArgs
                         return;
                     }
                 }
-                else if (status == MissionStatus.Aborted || status == MissionStatus.Cancelled || status == MissionStatus.Failed)
```

Comment:
Why do we no longer accept Aborted?

Reply:
I understand we might not want to send an error to the user in that case, but we also do want the robot to be unlocalized in that case.

Reply:
The reason is because there is no TaskStatus of type aborted, we could still check the mission status and if the mission has been aborted decide we want the robot to not be localized